src/modules/Dropdown/Dropdown.js

```diff
@@ -751,7 +751,7 @@ export default class Dropdown extends Component {
   renderText = () => {
     const { multiple, placeholder, search, text } = this.props
     const { searchQuery, value, open } = this.state
-    const hasValue = multiple ? !_.isEmpty(value) : !!value
+    const hasValue = multiple ? !_.isEmpty(value) : value !== ''
```

Comment:
It could also be that the tests need to be changed to match the new code.  Keep in mind that the tests are just there to assert that the code is doing what it should do.  So, if the code needs updating, it is likely that the tests need updating as well.